Source/MillicastPublisher/Public/IMillicastSource.h

```diff
@@ -2,11 +2,14 @@
 
 #pragma once
 
+//#include "api/media_stream_interface.h"
+#include "Containers/UnrealString.h"
+#include "Engine/TextureRenderTarget2D.h"
 #include "Misc/Optional.h"
-#include "api/media_stream_interface.h"
+#include "MillicastWebRTCInc.h"
```

Comment:
I added a new header for public WebRTC includes so we don't have our public header depending on our private headers.